sql/amo_prod/desktop_addons_by_client_v1/metadata.yaml

```diff
@@ -12,4 +12,4 @@ scheduling:
   dag_name: bqetl_amo_stats
   depends_on:
     - task_id: copy_deduplicate_main_ping
-      dag_name: main_summary
+      dag_name: copy_deduplicate
```

Comment:
Moved to `copy_deduplicate` in https://github.com/mozilla/telemetry-airflow/pull/1047